gnome-shell/upstream/theme/gnome-shell-sass/widgets/_login-lock.scss

```diff
@@ -51,25 +49,6 @@ $_gdm_dialog_width: 25em;
   }
 }
 
-.login-dialog-auth-menu-button-title {
-  @include fontsize($base_font_size - 1);
-  color: darken($_gdm_fg,40%);
-  font-weight: bold;
-  padding-top: $base_padding;
-  padding-bottom: $base_padding * 2;
-  padding-left: $base_padding * 2;
-  padding-right: $base_padding * 2;
-}
-
-.login-dialog-auth-menu-button-popup-menu-box {
-  padding: $base_padding * 3;
-}
-
-.login-dialog-auth-menu-button-item {
```

Comment:
We need to keep these changes as they are used by patches that we provide in ubuntu